src/hooks/useMinTradableAmountInOWL.ts

```diff
@@ -0,0 +1,14 @@
+import { useEffect } from 'react'
+import useSafeState from './useSafeState'
+import { dexPriceEstimatorApi } from 'api'
+import BigNumber from 'bignumber.js'
+
+export function useMinTradableAmountInOWL(networkId: number): null | BigNumber {
```

Comment:
out of curiousity - why no `useMemo` here instead of the effect? is it for the initial `null` render?

Reply:
How would I use `useMemo`? `dexPriceEstimatorApi.getMinOrderAmounInOWL` is async

Reply:
ah yes, true. didnt even notice that